src/seedu/addressbook/data/person/UniquePersonList.java

```diff
@@ -122,6 +117,23 @@ public void remove(ReadOnlyPerson toRemove) throws PersonNotFoundException {
         }
     }
 
+    /**
+     * Sort all persons in the list
+     */
+    public void sort() {
+        Collections.sort(internalList, new Comparator<Person>() {
+            @Override
+            public int compare(Person p1, Person p2) {
+                String name1 = p1.getName().toString();
+                String name2 = p2.getName().toString();
+                return name1.compareToIgnoreCase(name2);
+            }
+        });
+    }
+
```

Comment:
There seem to be one too many line breaks here. Try not to have too many as it might affect the readability of your code.